### DISCUSSION TITLE
Unknown authentication strategy "jwt" on A2Hosting

### DISCUSSION THREAD

**Discussion by fjrfac** [requarks/wiki#3398] [Help / Questions]
**Question**
Hello, I'm trying to make wikijs run on A2Hosing (shared) with no success.

I was able to successfully install it and to create the admin, but just right after that it gave me this error **Unknown authentication strategy "jwt"**.
When I reload the page it just page timeout. Does anyone have any suggestion?

**Host Info (please complete the following information):**
Shared hosting A2Hosting
Wiki.js version: 2.4.107
Node.js version 12.9.0
Database engine: PostgreSQL 9.6

**Comment by Smankusors** [comment]
Possible duplicate of #1822 

How long you waited before reload the page btw?

**Comment by fjrfac** [comment]
hello Smankusors, the error is similar to #1822 but in my case reloading the page doesn't solve it. Right now the website is not even loading you can check [here](https://support.dcs.film/). After I finished the installation process I got the JWT error, then I tried to visit the login page and it stopped loading the website.

**Comment by Smankusors** [comment]
How do you setup the Wiki.js? Can you post full logs of your Wiki.js?

**Comment by forsetius** [comment]
I can confirm the issue. I run it on Smarthost.pl hosting. It uses LiteSpeed as web server and Passenger to run the Node.

Wiki.js version: 2.4.107
Node.js version 12.9.0
Database engine: MariaDB 10.3

Also tried with Node.js 10.20.1 and 11.15.0 - with the same effect.

**Comment by axonasif** [comment]
So, nobody could figure out this yet?
I've literally wasted hours trying to get this work but no luck.
I tried thru `cPanel > Setup NodeJS App` with different configurations, including DB.
